docs/development/core/server/kibana-plugin-server.pluginsetupcontext.md

```diff
@@ -14,5 +14,5 @@ export interface PluginSetupContext
 
 |  Property | Type | Description |
 |  --- | --- | --- |
-|  [elasticsearch](./kibana-plugin-server.pluginsetupcontext.elasticsearch.md) | `{`<p/>`        adminClient$: Observable<ClusterClient>;`<p/>`        dataClient$: Observable<ClusterClient>;`<p/>`    }` |  |
+|  [elasticsearch](./kibana-plugin-server.pluginsetupcontext.elasticsearch.md) | <code>{`<p/>`        adminClient$: Observable&lt;ClusterClient&gt;;`<p/>`        dataClient$: Observable&lt;ClusterClient&gt;;`<p/>`    }</code> |  |

```

Comment:
ooooh. they fix one thing and break another. let's don't block this PR
![2019-04-02_22-07-22](https://user-images.githubusercontent.com/3198181/55432689-d98d9980-5593-11e9-892a-0fc112dd949b.png)